modules/tinymce/src/themes/silver/test/ts/browser/editor/sizing/ResizeTest.ts

```diff
@@ -11,13 +11,13 @@ UnitTest.asynctest('Editor resize test', (success, failure) => {
   Theme();
 
   TinyLoader.setup((editor: Editor, onSuccess, onFailure) => {
-    const cAssertEditorSize = (expectedWidth: number, expectedHeight: number) => Chain.control(
-      Chain.op((container: SugarElement) => {
+    const cAssertEditorSize = (expectedWidth: number, expectedHeight: number) =>
+      Chain.control(Chain.op((container: SugarElement) => {
```

Comment:
I find this indentation a little hard to read, maybe if we're going to bring down Chain.control we can also bring Chain.op down another level?

```javascript
    const cAssertEditorSize = (expectedWidth: number, expectedHeight: number) =>
      Chain.control(
        Chain.op((container: SugarElement) => {
          Assertions.assertEq(`Editor should be ${expectedHeight}px high`, expectedHeight, container.dom.offsetHeight);
          Assertions.assertEq(`Editor should be ${expectedWidth}px wide`, expectedWidth, container.dom.offsetWidth);
        }),
        Guard.addLogging('Ensure that the editor has resized')
      );
```

Reply:
I'd agree with James here, which is why I asked why this was originally changed as the original version was easier to read for me. If we don't want that, then I'd also be fine with the above suggestion.

Reply:
Ah, wrong correction, my bad. I should actually have looked at what change I made the first time, not guessed.
I'll properly change it back to the original, so we don't have to bother. I admit with James, but eh, not worth fussing about.